### PR TITLE
Card initial support

### PR DESCRIPTION
**Pull request by Brianggalvez**
Initial support for Card, currently supports setting ContentView

### CODE REVIEW

File: titanium-material.xcodeproj/xcuserdata/Brian.xcuserdatad/xcschemes/xcschememanagement.plist [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hansemannn) [inline titanium-material.xcodeproj/xcuserdata/Brian.xcuserdatad/xcschemes/xcschememanagement.plist:1]: Remove

File: Classes/TiMaterialCardViewProxy.m [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+#import "TiMaterialCardView.h"
+#import "TiUtils.h"
+
+@implementation TiMaterialCardViewProxy
+
+
+- (void)setContentView:(id)viewProxy
+{
+    ENSURE_TYPE(viewProxy, TiViewProxy);
+    [[(TiMaterialCardView *)[self view] cardView] setContentView:[viewProxy view]];
```
Comment (hansemannn) [inline Classes/TiMaterialCardViewProxy.m:18]: Does the layouting work properly? I would assume that some auto-sizing does not work out of the box.

File: Classes/TiMaterialCardView.m [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+//
+
+#import "TiMaterialCardView.h"
+
+@implementation TiMaterialCardView
+
+-(Card *)cardView
+{
+    if(_cardView == nil) {
+        _cardView = [[Card alloc]initWithFrame:self.bounds];
```
Comment (hansemannn) [inline Classes/TiMaterialCardView.m]: `[[Card alloc] initWithFrame: [super bounds]];`

File: Classes/TiMaterialCardView.m [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
```
Comment (hansemannn) [inline Classes/TiMaterialCardView.m]: Remove blank line

**Review by hansemannn — CHANGES_REQUESTED** [review]
Left a few review comments. Can you also add a reference to the readme and an example (if possible)?

**Comment by hansemannn** [comment]
Fixed the issues myself and will update some additional code snippets after merging. PR **approved**!

**Review by hansemannn — APPROVED** [review]

**Comment by hansemannn** [comment]
@Brianggalvez I just tested it and unfortunately it does not work in any case. Did you try your changes before submitting the PR? I may need to revert the merge.

**Comment by Brianggalvez** [comment]
Hi Hans, this is my working test code without your changes 
`var card = Material.createCardView({
  height: 200,
  width: 300,
  backgroundColor: "#eceded"
});
card.setContentView(Ti.UI.createView({
  height: 50,
  width: 50,
  backgroundColor: 'red'
}));
win.add(card);`
simulator screen shot - iphone 7 plus - 2018-04-25 at 12 30 13

**Comment by hansemannn** [comment]
Okay, then the sizing is indeed broken (with and without my changes). The width is not respected properly.

**Comment by Brianggalvez** [comment]
Exactly, but I'm not sure that even the card is fully working as it doesn't inherits either backgroundColor nor borderRadius as in the example there isn't any explicit set of these. [Sample](https://github.com/CosmicMind/Samples/blob/master/Projects/Programmatic/Card/Card/ViewController.swift)

**Comment by hansemannn** [comment]
Ok, so it seems like the width is determined by the card-view and the height by the content view. So the content view automatically fills the card view container as well. I have exposed the `backgroundColor` property, which sets the native color on the `Card` instead of the view proxy, enabling proper a corner radius handling (which is also already exposed). I will push an updated sample soon.

**Pull request merged**